src/index.html

```diff
@@ -10,13 +10,94 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
     <title>Moyo header</title>
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a href="index.html">
+        <img
+          src="images/logo.png"
+          alt="Логотип"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav_items">
+          <li class="nav_item">
+            <a
+              href="#Apple"
+              class="nav_link is-active"
+            >
+              APPLE
+            </a>
+          </li>
+          <li class="nav_item">
```

Comment:
Add the blank line between the links
```suggestion
          </li>
          
          <li class="nav_item">
```